android/flakey.txt

```diff
@@ -0,0 +1,79 @@
+# fully qualified  test name (c/n/p from report)  followed by option kernel versions followed by hardware platform
+# EBNF (roughly)
+#
+# Line
+# : STATE TESTNAME (KERNEL_VERSIONS) (HARDWARE) (ANDROID_VERSIONS)
+#
+# STATE
+# I | F | B | E
```

Comment:
Maybe explain what the different STATE's means?

Reply:
ok thank you @sumitsemwal , @liuyq do you know what B stands for?

Reply:
No, I don't know that.

Reply:
maybe we should drop it from the list?

Reply:
OK cool, but we are sure about the others? maybe we should explain them in a comment so we don't forget them moving forward?

Reply:
updated with comments on the meaning